inc/class-wpseo-utils.php

```diff
@@ -766,7 +766,7 @@ public static function get_plugin_name( $plugin ) {
 	 * @return string
 	 */
 	public static function get_site_name() {
-		return trim( strip_tags( get_bloginfo( 'name' ) ) );
+		return trim( wp_strip_all_tags( get_bloginfo( 'name' ), true ) );
```

Comment:
As `wp_strip_all_tags` already trims the content, this does not have to be wrapped again.